hyperjumptech/monika
# Docs: Update Monika Documentation

# Monika Pull Request (PR)  

## What feature/issue does this PR add  
1. This PR solves #381 

## How did you implement / how did you fix it  
1.  Improved Monika documentation according to the ticket

## How to test  
1. Run `cd docs && npm run dev`
2. Check if the content matches

## Additional information

Configuration examples in Installation section (which is now removed) is now in Guides > Notification
